### PR TITLE
Feature/perspectives ecosystem

### PR DESCRIPTION
**Pull request by sotous**
Finished perspective ecosystem data structure logic. Regarding https://github.com/uprtcl/js-uprtcl/issues/195.

### CODE REVIEW

File: src/services/data/data.service.ts [hunk 1]
```diff
@@ -1,19 +1,43 @@
...
   }
 
   async createData(
     data: Hashed<Object>, 
     _loggedUserId: string | null): Promise<string> {
 
     console.log('[UPRTCL-SERVICE] createData', data);
+
+    const dataCoded = {...data.object};
+    const castToObject = JSON.parse(JSON.stringify(dataCoded));
```
Comment (pepoospina) [inline src/services/data/data.service.ts:23]: Why do you need to both spread and parse? seems excessive. If the issue is with typescript complaining about the type of data.object, you can do a cast (data.object as any). Did you try that?

**Review by pepoospina — COMMENTED** [review]
great work!

File: src/services/data/data.service.ts (hunk 1 above)
Comment (sotous) [reply src/services/data/data.service.ts:23]: That's correct!, thanks for the review. I'll add the fix.

**Comment by sotous** [comment]
> great work!

Thank you! :+1:

**Pull request merged**